examples/vector-widget/VectorWidget.js

```diff
@@ -8,93 +8,72 @@
  */
 "use strict";
 
-var React = require('react');
-var ReactART = require('react-art');
-var Group = ReactART.Group;
-var Shape = ReactART.Shape;
-var Surface = ReactART.Surface;
-var Transform = ReactART.Transform;
-
-var MOUSE_UP_DRAG = 0.978;
-var MOUSE_DOWN_DRAG = 0.9;
-var MAX_VEL = 11;
-var CLICK_ACCEL = 3;
-var BASE_VEL = 0.15;
+const React = require('react');
```

Comment:
Can we split this file out into a separate PR? It's unclear if you needed to change something here to support the new renderer or if it is just a style change. Regardless it's unrelated.